repo/repo.go

```diff
@@ -9,6 +9,7 @@ import (
 	"strconv"
 
 	"github.com/rotsg/academy-go-q32021/model"
+	"github.com/rotsg/academy-go-q32021/universities"
 )
 
 type Repo struct {
```

Comment:
Please take care of names, as we saw in controller, ambiguity can be caused as there could not be two structs with this name. and when increasing this will cause confusion, this is considered a bad practice.